docs/examples/201/web-app-conditional-log/logging.bicep

```diff
@@ -0,0 +1,49 @@
+param logging bool = false
+param appName string
+param location string = resourceGroup().location
+
+var appInsightName = toLower('appi-${appName}')
+var logAnalyticsName = toLower('la-${appName}')
+
+resource appServiceAppSettings 'Microsoft.Web/sites/config@2020-06-01' = if (logging) {

```

Comment:
I don't think you need the `if (logging)` checks *in* the module, since the whole module will not be called unless the condition of the module is met in `main.bicep`. So you should be able to remove the logging param entirely

Reply:
I wondered this and actually did put it at the module level....just left it inside.  I did remove it and retest deployment.  Got the same result.

Code changes pushed up.

Reply:
Thanks for the help!